### DISCUSSION TITLE
provided disabled parameter: application id

### DISCUSSION THREAD

**Discussion by hmzdot** [reach-sh/reach-lang#926] [Q&A] [answered]
Lately there seems to be issue with indexer so that queries error with:
```js
{ message: "provided disabled parameter: application-id" }
```

**Comment by hmzdot** [comment]
Hamir's answer:
> I think someone else ran into a similar issue.
> I might be mistaken, but I think they got around it by using https://indexer.testnet.algoexplorerapi.io/... instead of https://algoindexer.testnet.algoexplorerapi.io/....

**Reply by iGavroche** [reply]
If we are talking about updating the domain prefix in the reachsdk js file this is the error we now get:

`Access to XMLHttpRequest at 'https://indexer.testnet.algoexplorerapi.io/v2/accounts/<Account Address>' from origin 'http://localhost:8080' has been blocked by CORS policy: Request header field x-indexer-api-token is not allowed by Access-Control-Allow-Headers in preflight response.`

**Reply by BakungaBronson** [reply]
Getting a similar error as well.

**Comment by Fraccas** [comment, marked as answer]
I had the same issue. For all of the projects I'm involved with we've started using AlgoNode for the API provider. 
```
reach.setWalletFallback(reach.walletFallback({
        providerEnv: {
          ALGO_TOKEN: '',
          ALGO_SERVER: "https://testnet-api.algonode.cloud",
          ALGO_PORT: '',
          ALGO_INDEXER_TOKEN: '',
          ALGO_INDEXER_SERVER: "https://testnet-idx.algonode.cloud",
          ALGO_INDEXER_PORT: '',
        }, MyAlgoConnect}));
 ```

**Reply by iGavroche** [reply]
this is perfect! Thank you @Fraccas !

**Reply by TibebeJS** [reply]
hello, thanks for your solution! I had 
```
 const stdlib = loadStdlib({
    REACH_CONNECTOR_MODE: "ALGO",
    REACH_DEBUG: "YES",
  });

  stdlib.walletFallback({
    // providerEnv: "TestNet",
    provider: {
      ALGO_TOKEN: "",
      ALGO_SERVER: "https://testnet-api.algonode.cloud",
      ALGO_PORT: "",
      ALGO_INDEXER_TOKEN: "",
      ALGO_INDEXER_SERVER: "https://testnet-idx.algonode.cloud",
      ALGO_INDEXER_PORT: "",
    },
  });
```
  
  but still kept facing the same issue, what could I be missing?

**Reply by hmzdot** [reply]
@TibebeJS in what environment are you running the stdlib? Is it something like a React app or node.js? 
You can also use new endpoints for the AlgoExplorer which starts with https://indexer.testnet.algoexplorerapi.io/ instead of https://algoindexer.testnet.algoexplorerapi.io/

**Reply by TibebeJS** [reply]
both nodejs and Vue

**Reply by hmzdot** [reply]
On the node.js part, setting a wallet fallback wouldn't make sense you should just do `setProvider`

**Reply by lorenzopolidori** [reply]
I have done the same, following a very similar issue with the Algorand indexer, see discussion thread on the Algorand discord https://discord.com/channels/491256308461207573/877184125654667285/973145663346991124

**Reply by DanBurton** [reply]
The reach stdlib has updated its Algorand settings for `providerEnv: 'TestNet'` and `providerEnv: 'MainNet'` to use AlgoNode. Thanks for the pointer!

**Reply by Thomas-Bibby** [reply]
I tried the solution above and also updated my stdlib to latest 0.1.10-rc.6 version and I still get the same issue. Any other ideas? I'm using React for my web interface btw. Thanks
